format/index.d.ts

```diff
@@ -0,0 +1,87 @@
+import { Store } from 'effector';
+
+export function format<A>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+): Store<string>;
+export function format<A, B>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+): Store<string>;
+export function format<A, B, C>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+): Store<string>;
+export function format<A, B, C, D>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+  d: Store<D>,
+): Store<string>;
+export function format<A, B, C, D, E>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+  d: Store<D>,
+  e: Store<E>,
+): Store<string>;
+export function format<A, B, C, D, E, F>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+  d: Store<D>,
+  e: Store<E>,
+  f: Store<F>,
+): Store<string>;
+export function format<A, B, C, D, E, F, G>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+  d: Store<D>,
+  e: Store<E>,
+  f: Store<F>,
+  g: Store<G>,
+): Store<string>;
+export function format<A, B, C, D, E, F, G, H>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+  d: Store<D>,
+  e: Store<E>,
+  f: Store<F>,
+  g: Store<G>,
+  h: Store<H>,
+): Store<string>;
+export function format<A, B, C, D, E, F, G, H, I>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+  d: Store<D>,
+  e: Store<E>,
+  f: Store<F>,
+  g: Store<G>,
+  h: Store<H>,
+  i: Store<I>,
+): Store<string>;
+export function format<A, B, C, D, E, F, G, H, I, J>(
+  string: TemplateStringsArray,
+  a: Store<A>,
+  b: Store<B>,
+  c: Store<C>,
+  d: Store<D>,
+  e: Store<E>,
+  f: Store<F>,
+  g: Store<G>,
+  h: Store<H>,
+  i: Store<I>,
+  j: Store<J>,
+): Store<string>;
```

Comment:
```suggestion
export function format<T extends Array<Store<any>>>(
  string: TemplateStringsArray,
  ...stores: T,
): Store<string>;
```

[Try it](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgZRtApgGjgYyugQxnVQzgF84AzKCEOAcnSqvRzSgYCguATNgDYF81AK4A7dsAjjq0EEQA8AQTjoAHsXG8AznGVQoBAJ6LS+RQXHGAfHYAUXOHB0wowcQHMAXHAAq6OBCxKjuXjoGRsaYTnAAdAmuGDq+yjEAlL7m6IquYZ42ANw8ODKucARwALy4+EQkHOj2AIzpXKXi5QBG1bWEIY32AERUEBBDbe1l8Pg6ogLwNaNQCjAABgBiY3AAJAgElHtd5Gs8QA)